test/symdiff-jade.test.js

```diff
@@ -1,4 +1,19 @@
-var extract = require('../index');
+var extract  = require('../index'),
+    readFile = require('fs').readFile,
```

Comment:
Would you mind using `readFileSync` here? These tests are so small that speed doesn’t matter yet and I find synchronous code easier to read.